data-model/src/main/java/io/micronaut/data/model/query/builder/sql/SqlQueryBuilder.java

```diff
@@ -787,14 +787,19 @@ protected void selectAllColumns(QueryState queryState, StringBuilder queryBuffer
                         } else {
                             columnName = asPath(propertyAssociations, prop);
                         }
+                        String columnAlias = prop.getAnnotationMetadata().stringValue(MappedProperty.class, MappedProperty.ALIAS).orElse("");
```

Comment:
Personally, I would use `orElse(null)` and a null check

Reply:
I thought to use it, but this covers case if someone does this (forgets to remove or whatever) 

```
@MappedProperty(alias = "")
```

so this is maybe simpler/safer

Reply:
That might trigger the default value and return empty, but not sure